python/lsst/ctrl/bps/bps_config.py

```diff
@@ -78,6 +78,16 @@ def __init__(self, other, search_order=None):
         # using the inherited update() method which does not rely on super
         # class __getitem__ method.
         super().__init__()
+
+        if isinstance(other, str):
+            # First load default config from ctrl_bps, then
+            # override with user config.
+            bps_defaults = realpath(dirname(__file__) + "/../../../../etc/bps_defaults.yaml")
```

Comment:
Can we use  `os.path.join()` to concatenate the paths instead? It would lower the risk of running into problems due to extra (or missing) backslashes.

Reply:
Wasn't happy with the hardcoded relative path.  @timj pointed me to pkg_resources and from there I ended up with importlib.resources.  To use this, etc/bps_defaults.yaml will need to be moved down inside python/lsst/ctrl/bps.